content/docs/dvclive/how-it-works.md

```diff
@@ -14,17 +14,19 @@ The contents of the directory will depend on the methods used:
 | [live.log_sklearn_plot](/doc/dvclive/api-reference/live/log_sklearn_plot) | `dvclive/plots/sklearn`    |
 | [live.make_report](/doc/dvclive/api-reference/live/make_report)           | `dvclive/report.{md/html}` |
 | [live.make_summary](/doc/dvclive/api-reference/live/make_summary)         | `dvclive/metrics.json`     |
+| [make_dvcyaml](/doc/dvclive/api-reference/live/end)                       | `dvclive/dvc.yaml`         |
```

Comment:
```suggestion
| [live.next_step](/doc/dvclive/api-reference/live/next_step )         | `dvclive/metrics.json`<br>`dvclive/report.{md/html}`<br>`dvclive/dvc.yaml` (if `save_dvc_exp=True`)    |
| [live.end](/doc/dvclive/api-reference/live/end )         | `dvclive/metrics.json`<br>`dvclive/report.{md/html}`<br>`dvclive/dvc.yaml` (if `save_dvc_exp=True`)    |
```

What do you think? We could drop the `make_dvcyaml` line and the admon below.

Reply:
> What do you think? 

Great idea